### PR TITLE
New feature - flac converter

### PR DESCRIPTION
**Pull request by aszweryn**
A script converting various audio files into those with .flac extensions. Its bit depth is set to the standard 16 bits. I wanted to learn a bit more, thus I added an optional 'funny' part. Should you accept all features, or only the essential one, I'll also update README file.
PS. The requirements for the 'funny' part are the following packages: cowsay, lolcat

### CODE REVIEW

File: audiotoolbox.sh [hunk 1]
```diff
@@ -37,4 +37,20 @@ audioclip(){
 	rm "temp_$1"
 }
 
-
+flacer(){
+	echo Do you want a funny version? [y/N]
```
Comment (Arthur-Sebastian) [inline audiotoolbox.sh]: Not informative. Gets in the way of user comfort, is asked every time. If you want to add such configuration option I'd recommend implementing a config file.

File: audiotoolbox.sh [hunk 2]
```diff
@@ -37,4 +37,20 @@ audioclip(){
...
+	if [ "$opt" == "N" ]; then
+	ffmpeg -hide_banner -i "$1" -c:a flac -af aformat=s16:44100 "${1%.*}.flac"
+	echo -e "\e[1;33mConversion complete: "${1%.*}.flac" in $PWD\e[0m"
+	fi
+	
+	if [ "$opt" == "y" ]; then
+	NAME=${1%.*}
+	ffmpeg -hide_banner -i "$1" -c:a flac -af aformat=s16:44100 "${1%.*}.flac"
+	if [[ -f $NAME.flac ]]; then
+	cowsay -f dragon "Conversion complete: $NAME.flac in $PWD" | lolcat
```
Comment (Arthur-Sebastian) [inline audiotoolbox.sh]: To be removed. Adds three big dependencies (lolcat also requires ruby) which are not essential for the script's functionality, and also slow it down heavily on weaker machines.

File: audiotoolbox.sh [hunk 3]
```diff
@@ -37,4 +37,20 @@ audioclip(){
...
 }
 
-
+flacer(){
+	echo Do you want a funny version? [y/N]
+	read $opt;
+	
+	if [ "$opt" == "N" ]; then
+	ffmpeg -hide_banner -i "$1" -c:a flac -af aformat=s16:44100 "${1%.*}.flac"
+	echo -e "\e[1;33mConversion complete: "${1%.*}.flac" in $PWD\e[0m"
```
Comment (Arthur-Sebastian) [inline audiotoolbox.sh]: Repeated message that is displayed regardless of process exit code, recommended to move it outside the conditional statement blocks, near the function closing bracket.

**Review by Arthur-Sebastian — CHANGES_REQUESTED** [review]
A few changes need to be made, see the comments.

File: audiotoolbox.sh (hunk 1 above)
Comment (aszweryn) [reply audiotoolbox.sh]: That's true, will do.

File: audiotoolbox.sh (hunk 3 above)
Comment (aszweryn) [reply audiotoolbox.sh]: This problem will be resolved jointly with the removal of the parts that require lolcat.

File: audiotoolbox.sh (hunk 2 above)
Comment (aszweryn) [reply audiotoolbox.sh]: Understood.

**Review by Arthur-Sebastian — APPROVED** [review]
Changes implemented, no further comments are required.

**Pull request merged**